### PR TITLE
Add docs for Amplitude session middleware

### PR DESCRIPTION
**Pull request by prayansh**
### Proposed changes
Adding docs for the new Amplitude session id middleware that enables Amplitude Destination Actions

### Merge timing
- ASAP once approved

### Related issues (optional)
https://github.com/segment-integrations/analytics-android-integration-amplitude/pull/59

### CODE REVIEW

**Review by bsneed — APPROVED** [review]

**Review by forstisabella — APPROVED** [review]

File: src/connections/destinations/catalog/actions-amplitude/index.md [hunk 1]
```diff
@@ -78,6 +78,34 @@ To enable session tracking in Amplitude when using the [Segment Kotlin library](
    analytics.add(AmplitudeSession())
    ```
 
+### Enable Amplitude session tracking for iOS
+
+To enable session tracking in Amplitude when using the [Segment iOS library](https://github.com/segmentio/analytics-ios):
+1. Add the [Amplitude Session plugin](https://github.com/segment-integrations/analytics-ios-integration-amplitude/blob/amplitude-session/Pod/Classes/SEGAmplitudeSession.m) to your project.
```
Comment (migs647) [inline src/connections/destinations/catalog/actions-amplitude/index.md:84]: This isn't considered a plugin. We should clean up the language and call it middleware.

File: src/connections/destinations/catalog/actions-amplitude/index.md [hunk 2]
```diff
@@ -78,6 +78,34 @@ To enable session tracking in Amplitude when using the [Segment Kotlin library](
...
+   	SEGAnalyticsConfiguration *configuration = [SEGAnalyticsConfiguration configurationWithWriteKey:SEGMENT_WRITE_KEY];
+   	configuration.trackApplicationLifecycleEvents = true
+   	configuration.sourceMiddleware = @[[[SEGAmplitudeSession alloc] init]];
+   	[SEGAnalytics setupWithConfiguration:configuration];
+   ```
+
+### Enable Amplitude session tracking for Android
+
+To enable session tracking in Amplitude when using the [Segment Android library](https://github.com/segmentio/analytics-android):
+1. Add the [Amplitude Session plugin](https://github.com/segment-integrations/analytics-android-integration-amplitude/blob/master/src/main/java/com/segment/analytics/android/integrations/amplitude/AmplitudeSessionId.java) to your project.
```
Comment (migs647) [inline src/connections/destinations/catalog/actions-amplitude/index.md:97]: This isn't considered a plugin. We should clean up the language and call it middleware.

**Review by migs647 — COMMENTED** [review]
Lets change it from plugin (since this isn't a plugin architecture) to middleware for ios / android.

File: src/connections/destinations/catalog/actions-amplitude/index.md (hunk 1 above)
Comment (prayansh) [reply src/connections/destinations/catalog/actions-amplitude/index.md:84]: good catch

**Pull request merged**